moped-database/migrations/1673559416128_remove_empty_strings_and_0/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Remove empty strings and 0 that we added when a user updates the moped_proj_funding table
+UPDATE moped_proj_funding SET fund_dept_unit = null WHERE fund_dept_unit = '';
+UPDATE moped_proj_funding SET funding_description = null WHERE funding_description = '';
+UPDATE moped_proj_funding SET funding_program_id = null WHERE funding_program_id = 0;
```

Comment:
thank you for adding this data cleanup! if we're going to run this migration, we should also remove that blank `fund_program` record with id `0` - otherwise the data will keep getting crufty.

Reply:
added!